pupa/tests/scrape/test_people_org_scrape.py

```diff
@@ -143,3 +142,12 @@ def test_committee_add_member_name():
     assert get_pseudo_id(c._related[0].person_id) == {'name': 'John Adams'}
     assert c._related[0].organization_id == c._id
     assert c._related[0].role == 'member'
+
+def test_person_add_membership_name():
+    p = Person('Leonardo DiCaprio')
+    p.add_membership('Academy of Motion Picture Arts and Sciences', role='winner', start_date='2016')
+    p._related[0].validate()
```

Comment:
Why is this line necessary? There's no similar line in the `test_committee_add_member_name` test.